extensions/cornerstone/src/initMeasurementService.ts

```diff
@@ -350,7 +351,13 @@ const connectMeasurementServiceToTools = (
       if (source.name !== CORNERSTONE_3D_TOOLS_SOURCE_NAME) {
         continue;
       }
+      const removedAnnotation = annotation.state.getAnnotation(uid);
       removeAnnotation(uid);
+      triggerCreateAnnotationMemo({
```

Comment:
how about a command in commandModule and more generic that annotation maybe? or it has modes, annotation, segmentation etc in future

Reply:
I added it to the command modules but I think it's best if we leave separate methods for different types. Even the API of the labelmap segmentation createMemo is different, it takes different arguments such as a voxelManager.